qiskit/dagcircuit/dagdependency.py

```diff
@@ -48,7 +48,7 @@
 
 
 class DAGDependency:
-    """Object to represent a quantum circuit as a directed acyclic graph
+    """Object to represent a quantum circuit as a DAG
```

Comment:
Personally I think using the full text here was clearer, but I don't feel strongly.

Reply:
done in 16b75c78ec152b33f79ceb47fd2016deb7a18559